src/reactviews/pages/QueryResult/commandBar.tsx

```diff
@@ -56,8 +62,39 @@ const CommandBar = (props: CommandBarProps) => {
         });
     };
 
+    const hasMultipleResults =
+        context.state.resultSetSummaries &&
+        Object.keys(context.state.resultSetSummaries).length > 1;
+
     return (
         <div className={classes.commandBar}>
+            {hasMultipleResults && (
+                <Button
+                    appearance="subtle"
+                    onClick={() => {
+                        maxView
+                            ? props.restoreResults?.()
+                            : props.maximizeResults?.();
+                        setMaxView((prev) => !prev); // Toggle maxView state
+                    }}
+                    icon={
+                        <img
```

Comment:
Simplify this with the `@fluentui/react-icons` library: https://github.com/microsoft/vscode-mssql/blob/main/src/reactviews/pages/ConnectionDialog/connectionsListContainer.tsx#L113C1-L117C19